python/triton_kernels/tests/test_mxfp.py

```diff
@@ -146,8 +146,8 @@ def test_mxfp_casting(
     if is_hip():
         if swizzle_value is not None or swizzle_scale is not None:
             pytest.skip("Other swizzling patterns are not supported by AMD GPU")
-        if quant_dtype == 'float8_e4m3fn':
-            pytest.skip("float8_e4m3fn cast hasn't been fully tested on AMD GPU")
+        if quant_dtype == 'float8_e4m3fn' and is_hip_cdna3():
+            pytest.skip("float8_e4m3fn cast hasn't been fully tested on AMD CDNA3")
```

Comment:
Does it fail on CDNA3 now? Have you tried? Maybe just drop it and see how CI bots think?

Reply:
Actually maybe doesn't make sense--given these types are not native so emulated on CDNA3.

Reply:
The test comes from Ravil's commit. I didn't run it on all the platform myself. But yeah when I looked at the code, I remember the emulation of non-native types are still getting improved.

Reply:
Just checked on mi300. Some of them still failed. Made it more fine-grained.

Reply:
Just checked on mi300. Those tests still fail.